src/pages/Appointments.tsx

```diff
@@ -230,7 +230,7 @@ function AppointmentCard(props) {
             <LinkToVideo onClick={() => !hideActions && toggleWebinar()} status={status} />
           ) : (
             <Typography sx={{ fontSize: "0.825rem", color: "grey.900" }}>
-              {appointment.location}
+              {appointment.location ? appointment.location : "Location not specified"}
```

Comment:
Nitpick, up to you 😛 
```suggestion
              {appointment.location ?? "Location not specified"}
```

Reply:
That doesn't work unfortunately